### PR TITLE
[1LP][RFR] Adding Multi-Region FA requirement and appropriate tests

### PR DESCRIPTION
**Pull request by izapolsk**
Multi-Region FA.

This is first PR is the chain of PRs with tests related to Multi-Region/Central Administration FAs

### CODE REVIEW

**Comment by izapolsk** [comment]
@john-dupuy, I can add it if you think it is necessary. from my point, it isn't crucial since it doesn't reduce readability or affect any .rules

File: cfme/test_requirements.py [hunk 1]
```diff
@@ -420,3 +420,9 @@ def test_quota_alert():
     description='Support of VMWare in MIQ/CFME',
     assignee_id='kkulkarn'
 )
+
+multi_region = pytest.mark.requirement(
```
Comment (mshriver) [inline cfme/test_requirements.py]: This file is _roughly_ alphabetical, mind putting this in the middle of the list? (`multi-tenancy` is out of place too if you want to move it...)

File: cfme/tests/cloud_infra_common/test_power_control_manual.py [hunk 2]
```diff
@@ -254,7 +260,6 @@ def test_power_operations_from_global_region():
             5.
             6. Power operations applicable for this vm should be working as expected
     """
-    pass
```
Comment (mshriver) [inline cfme/tests/cloud_infra_common/test_power_control_manual.py]: I think I would like to require this `pass` statement on manual test cases

File: cfme/tests/cloud_infra_common/test_retirement.py [hunk 3]
```diff
@@ -294,3 +296,27 @@ def test_resume_retired_instance(retire_vm, provider, remove_date):
 
     verify_retirement_date(retire_vm, expected_date=retire_date if retire_date else 'Never')
     assert retire_vm.is_retired is False
+
+
+@pytest.mark.manual('manualonly')
```
Comment (mshriver) [inline cfme/tests/cloud_infra_common/test_retirement.py]: We can't automate this?

Comment (izapolsk) [reply cfme/tests/cloud_infra_common/test_retirement.py]: good point, removing. everything in my FA is gonna be automated.

File: cfme/tests/cloud_infra_common/test_power_control_manual.py (hunk 2 above)
Comment (izapolsk) [reply cfme/tests/cloud_infra_common/test_power_control_manual.py]: since I have two votes against I obey :)

**Pull request merged**